dataload/owl2json/src/main/java/uk/ac/ebi/owl2json/xrefs/Bioregistry.java

```diff
@@ -92,6 +106,16 @@ public String getUrlForId(String databaseId, String id) {
             
         return uriFormat.getAsString().replace("$1", id);
     }
+    
+    private static String norm(String s) {
+        // see https://github.com/biopragmatics/bioregistry/blob/a7424ef4a0d22eaca61d3a86c6175e2059e9c855/src/bioregistry/utils.py#L128-L133
+        s = s.toLowerCase(Locale.ROOT);
+        s = s.replace(".", "");
+        s = s.replace("-", "");
+        s = s.replace("_", "");
+        s = s.replace("/", "");
```

Comment:
All these can be replaces in one operation, using s = s.replaceAll("[-_/.]","");